### PR TITLE
Introduce pino logger

### PR DESCRIPTION
**Pull request by andreachild**
Replaced file and console logging with the pino logger library for easier maintainability and future customization. Changed the `quiet` option to set the console log level to 'warn' if enabled ('debug' if disabled) instead of checking for the quiet value before each log. File logging is by default at the 'debug' level but could be customized in the future with a user-provided level. Removed some ARNs and other info from logged statements for privacy concerns.

Note: Removal of additional sensitive-leaning data will be done as a separate PR

### CODE REVIEW

File: src/NeptuneSchema.js [hunk 1]
```diff
@@ -271,26 +255,11 @@ async function getNodesProperties() {
...
 async function checkEdgeDirectionCardinality(d) {
-    let parameters = `{"from": "${d.from}", "edge": "${d.edge.label}", "to": "${d.to}"}`;
-    let queryFrom = `MATCH(from)-[r]->(to)
-                    WHERE $from in labels(from)
-                        AND type(r) = $edge
-                        AND $to in labels(to)
-                    WITH to, count(from) as rels
-                    WHERE rels > 1
-                    RETURN rels LIMIT 1`;
-    loggerLog(`Checking edge direction cardinality: ${queryFrom}`);
```
Comment (Cole-Greer) [inline src/NeptuneSchema.js]: Is this log dropped intentionally?

File: src/NeptuneSchema.js [hunk 2]
```diff
@@ -271,26 +255,11 @@ async function getNodesProperties() {
...
+    let responseFrom = await queryNeptune(queryFrom);
     let resultFrom = responseFrom.results[0];
-    let queryTo = `MATCH(from)-[r]->(to)
-                    WHERE $from in labels(from)
-                        AND type(r) = $edge
-                        AND $to in labels(to)
-                    WITH from, count(to) as rels
-                    WHERE rels > 1
-                    RETURN rels LIMIT 1`;
-    loggerLog(`Checking edge direction cardinality: ${queryTo}`);
```
Comment (Cole-Greer) [inline src/NeptuneSchema.js]: Same as above

**Review by Cole-Greer — APPROVED** [review]
LGTM, only one minor question.

File: src/NeptuneSchema.js (hunk 1 above)
Comment (andreachild) [reply src/NeptuneSchema.js]: Not intentional, added back a generic log message.

File: src/NeptuneSchema.js (hunk 2 above)
Comment (andreachild) [reply src/NeptuneSchema.js]: Not intentional, added back a generic log message.

**Pull request merged**